react/typings/globals.d.ts

```diff
@@ -11,6 +11,8 @@ declare global {
     showNavigationArrows: 'mobileOnly' | 'desktopOnly' | 'always' | 'never'
     showPaginationDots: 'mobileOnly' | 'desktopOnly' | 'always' | 'never'
     usePagination: boolean
+    fullWidth: boolean
+    arrowIconSize: number
```

Comment:
Does it make sense to make this prop Responsive?

Reply:
I thought about it and could not make up my mind.
I guess it would be useful to make it responsive since you might use larger arrows in larger screens

Reply:
Done ;)

Reply:
Done